utils/scripts/testconfig/test-variables.yaml

```diff
@@ -12,6 +12,18 @@ OCP_LDAP_USER:
   AUTH_TYPE: OCP_LDAP_USER_AUTH_TYPE
   USERNAME: OCP_LDAP_USER_USERNAME
   PASSWORD: OCP_LDAP_USER_PASSWORD
+OCP_LDAP_PROJECT_ADM:
```

Comment:
Maybe for clearing the code and make it more understandable, we should change the OCP_LDAP_USER to be OCP_LDAP_CLUSTER_ADM?

Reply:
These fields are just for reference and will be updated during runtime on jenkins or by the user while execution. IMHO, this is fine. Please let me know if it still needs to be changed.